Source/WebCore/rendering/svg/SVGPathData.h

```diff
@@ -26,4 +26,6 @@ class Path;
 
 Path pathFromGraphicsElement(const SVGElement*);
 
+const Path& sharedClipAllPath();
```

Comment:
What's the purpose of that?

Reply:
The comment says "The SVG specification wants us to clip everything, if clip-path doesn't have a child."
I moved it into RenderSVGResourceClipper.cpp since that seems to be the only user.